portal/src/graphql/portal/AcceptAdminInvitationScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback, useMemo } from "react";
+import { Text } from "@fluentui/react";
+import { FormattedMessage } from "@oursky/react-messageformat";
+import { useLocation, useNavigate } from "react-router-dom";
+import cn from "classnames";
+
+import { useAcceptCollaboratorInvitationMutation } from "./mutations/acceptCollaboratorInvitationMutation";
+import ButtonWithLoading from "../../ButtonWithLoading";
+import ShowError from "../../ShowError";
+
+import styles from "./AcceptAdminInvitationScreen.module.scss";
+
+const AcceptAdminInvitationScreen: React.FC = function AcceptAdminInvitationScreen() {
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const invitationCode = useMemo(() => {
+    return new URLSearchParams(location.search).get("code");
+  }, [location]);
+
+  const {
+    acceptCollaboratorInvitation,
+    loading,
+    error,
+  } = useAcceptCollaboratorInvitationMutation();
+
+  const onAccept = useCallback(() => {
+    acceptCollaboratorInvitation(invitationCode ?? "")
+      .then((appID) => {
+        if (appID !== null) {
+          navigate("../../");
+        }
+      })
+      .catch(() => {});
+  }, [acceptCollaboratorInvitation, invitationCode, navigate]);
+
+  return (
+    <main className={cn(styles.root, { [styles.loading]: loading })}>
+      {error && <ShowError error={error} />}
```

Comment:
The error thrown by backend should contain a `reason` field? I think we should handle the error and show localized error message.